src/Devtools.tsx

```diff
@@ -138,7 +147,17 @@ export const QueryRow: Component<{ query: Query }> = (props) => {
   return (
     <Show when={queryState()} keyed>
       {(queryState) => (
-        <div class={styles.queryRow}>
+        <div
+          onClick={() =>
+            setSelectedQueryHash(
+              props.query.queryHash === selectedQueryHash() ? null : props.query.queryHash,
+            )
+          }
+          classList={{
+            [styles.queryRow]: true,
+            [styles.selectedQueryRow]: selectedQueryHash() === props.query.queryHash,
+          }}
+        >
```

Comment:
If we're adding interactivity by adding click handlers then we should make it a button :D

Reply:
https://github.com/ardeora/solid-query-devtools/pull/2/commits/90f9e393b426ab1af22621f211c0031e3f76370c